.gitlab-ci.yml

```diff
@@ -58,7 +58,7 @@ default:
     name: parity-build
 
 .docker-env:                       &docker-env
-  image:                           parity/rust-builder:latest
+  image:                           paritytech/substrate-ci-linux:production
```

Comment:
Devhub's [node-template](https://github.com/substrate-developer-hub/substrate-node-template/blob/master/docker-compose.yml#L6) now use `parity/rust-builder:latest` as a base image when run in docker. Should I change it to the new image?

Reply:
I will make a PR there, thanks for the heads up. I know some other projects are using this image and will make sure all of them will use a new one.